### ISSUE TITLE
"nan" loss

### ISSUE THREAD

**Issue by tanshuai0219** [ShihaoZhaoZSH/LaVi-Bridge#13]
when I test llama2+transformer, I always get a nan loss after few hundred steps.
Could you give me some advices?

**Comment by tanshuai0219** [comment]
> when I test llama2+transformer, I always get a nan loss after few hundred steps. Could you give me some advices?

Is it critical to set the learning rate？

**Comment by ShihaoZhaoZSH** [comment]
Yes. We recommend using a relatively higher learning rate, such as 1e-4, for training the transformer models,
